src/Monolog/Logger.php

```diff
@@ -176,6 +176,25 @@ public function popHandler()
     }
 
     /**
+     * Set handlers, removing all existing ones.
+     * Falsey values will be ignored, and if a map is passed, keys will be ignored.
+     *
+     * @param array $handlers All elements must be of type HandlerInterface
+     * @return $this
+     */
+    public function setHandlers(array $handlers)
+    {
+        $this->handlers = array();
+        foreach($handlers as $handler) {
```

Comment:
Thanks, @camspiers. I'll amend the squash 2 fixes into the same commit.

Do you have a linter that picks up on this? I'm using phpcs and it didn't warn me of these.